convergedtarkus/semver_parser
# Allow using the backspace key in the example app

## Problem
The example app is hard to use as you cannot delete anything you type.

## Solution
Add support for the backspace key. This allows deleting one character at a time.

## Implementation Notes
Pretty straight forward. Backspace is a single byte and just requires removing the last entry in the input bytes being assembled.

## Tests Added
No, example app only.

## Testing Instructions
Use the example app and ensure the backspace key works. It works on historical commands, but those changes will not be saved to history (unless the command is run, in which case it is a new history entry).
